src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.SystemIAccessibleWrapper.cs

```diff
@@ -0,0 +1,249 @@
+
+#nullable disable
+
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using Accessibility;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class AccessibleObject
+    {
+        /// <summary>
+        /// This is a helper class that is used to wrap system IAccessible object
+        /// and to perform calls to system accessibility methods with check wrapped
+        /// IAccessible object for null and with COM exception handling to prevent
+        /// application crash in case system IAccessible object is not found.
+        /// </summary>
+        private class SystemIAccessibleWrapper : IAccessible
+        {
+            private IAccessible _systemIAccessible;
+
+            public SystemIAccessibleWrapper(IAccessible systemIAccessible)
+            {
+                _systemIAccessible = systemIAccessible;
+            }
+
+            public void accSelect(int flagsSelect, object varChild)
+            {
+                Execute(() => _systemIAccessible.accSelect(flagsSelect, varChild));
+            }
+
+            public void accLocation(out int pxLeft, out int pyTop, out int pcxWidth, out int pcyHeight, object varChild)
+            {
+                int left = 0;
+                int top = 0;
+                int width = 0;
+                int height = 0;
+
+                Execute(() =>
+                {
+                    _systemIAccessible.accLocation(out left, out top, out width, out height, varChild);
+
+                    Debug.WriteLineIf(CompModSwitches.MSAA.TraceInfo, "AccessibleObject.AccLocation: Setting " +
+                        left.ToString(CultureInfo.InvariantCulture) + ", " +
+                        top.ToString(CultureInfo.InvariantCulture) + ", " +
+                        width.ToString(CultureInfo.InvariantCulture) + ", " +
+                        height.ToString(CultureInfo.InvariantCulture));
+                });
+
+                pxLeft = left;
+                pyTop = top;
+                pcxWidth = width;
+                pcyHeight = height;
+            }
+
+            public object accNavigate(int navDir, object varStart)
+            {
+                return GetValue(() => _systemIAccessible.accNavigate(navDir, varStart));
+            }
+
+            public object accHitTest(int xLeft, int yTop)
+            {
+                return GetValue(() => _systemIAccessible.accHitTest(xLeft, yTop));
+            }
+
+            public void accDoDefaultAction(object varChild)
+            {
+                Execute(() => _systemIAccessible.accDoDefaultAction(varChild));
+            }
+
+            public object accParent => GetValue(() => _systemIAccessible.accParent);
+
+            public int accChildCount => GetStructValue(() => _systemIAccessible.accChildCount);
+
+            public object get_accChild(object childID) =>
```

Comment:
If wrap, we keep the arrow on the next line
```suggestion
            public object get_accChild(object childID) 
                => GetValue(() => _systemIAccessible.get_accChild(childID));
```

Reply:
Fixed. Thanks!